test/method/testbader.py

```diff
@@ -56,3 +57,31 @@ def test_val(self):
         refData = [6.9378, 0.3639, 0.3827]  # values from `bader` package
 
         assert_allclose(self.analysis.fragcharges, refData, atol=0.15)
+
+    def test_chgsum_hf(self):
+        """Does the sum of charges equate to the number of electrons for a simple molecule?"""
+
+        hfpath = os.path.dirname(os.path.realpath(__file__)) + "/hf.out"
```

Comment:
Prefer `os.path.join` rather than hard-coding the separator `/` (If this happened in other PRs, I missed it.)

Reply:
Thanks for letting me know of this. I've just change this to use `os.path.join`.